extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -137,7 +141,13 @@ export class PublishOptionsDialog {
 	* Ok button click, will update the deployment options with selections
 	*/
 	protected execute(): void {
+		if (this.resetClicked) {
+			// reset optionsvalueNameLookup with fresh deployment options
+			this.optionsModel.setOptionsToValueNameLookup();
```

Comment:
I'm confused by this change and comment. If someone clicks reset, then changes a couple options before clicking "Ok", then their changes will go away according to this comment. There shouldn't need to be any reset specific logic in `execute()`